src/view3d.cpp

```diff
@@ -355,18 +518,18 @@ void View3D::drawMeshes(const TransformState& transState,
         meshFaceShader.setUniformValue("lightDir_eye", lightDir.x, lightDir.y, lightDir.z);
         for (size_t i = 0; i < geoms.size(); ++i)
             geoms[i]->drawFaces(meshFaceShader, transState);
-        meshFaceShader.release();
+        // meshFaceShader.release();
```

Comment:
Is this unnecessary?  If so, let's remove it entirely rather than commenting it out.